### ISSUE TITLE
Error when boot_diagnostics is false

### ISSUE THREAD

**Issue by blakelead** [scalair/terraform-azure-vm#2]
If boot_diagnostics is false and sa_name is empty, the following error occurs:

```
Error: compute.VirtualMachinesClient#CreateOrUpdate: Failure sending request: StatusCode=400 -- Original Error: Code="BadRequest" Message="Invalid URI: The hostname could not be parsed." Target="vm.properties.diagnosticsProfile.bootDiagnostics.storageUri"
```

**Comment by Xat59** [comment]
This commit fixes it : 52343e4 .

**Comment by Xat59** [comment]
The PR https://github.com/scalair/terraform-azure-vm/pull/4 has been merged.
